src/intl/messages.js

```diff
@@ -512,12 +512,20 @@ export const messages: { [messageId: string]: MessageType } = {
   operatingSystemInstalledOnVm: 'Operating system installed on the virtual machine.',
   optimizedFor: 'Optimized For',
   optionalUserDescriptionOfVm: 'Optional user description of the virtual machine.',
+  optionIsNotSavedOnTheServer: {
+    message: 'This option is not saved on the server. {persistenceReEnableHowTo}',
+    description: 'Inform the user that server-side persistence is not enabled for this option. If it can be re-enabled then provide translated instructions (parameter persistenceReEnableHowTo). Otherwise leave it empty.',
```

Comment:
@rszwajko Why did you split the message into 2 parts `optionIsNotSavedOnTheServer` and  `persistenceReEnableHowTo` and what do you mean by "If it can be re-enabled then provide translated instructions". Is there a use case in which it can't be re-enabled?

Reply:
I would like to re-use the `optionIsNotSavedOnTheServer` message for all future options and some of them might be local only (i.e. like notifications today).